docs/source/api/deliveryservices_id_servers.rst

```diff
@@ -23,7 +23,7 @@
 
 ``GET``
 =======
-Retrieves properties of Edge-Tier servers assigned to a :term:`Delivery Service`.
+Retrieves properties of Edge-Tier servers assigned to a :term:`Delivery Service`. This includes servers explicitly assigned to a Delivery Service and servers assigned via a :term:`Cache Assignment Group`.
```

Comment:
If you're going to make `Cache Assignment Group` a `:term:` - and I think you should - then you need to add a definition for it to the Glossary. Otherwise this does nothing except generate a compilation warning.

Also, terms should be linked to the glossary (via `:term:`) at least once per major section ("page" in an HTML context), so you'd need to do that on all the other "pages" where you mentioned CAGs before.